src/component/context/ContectFormChip/ContextFormChip.tsx

```diff
@@ -0,0 +1,30 @@
+import { useStyles } from 'component/context/ContectFormChip/ContextFormChip.styles';
+import { Cancel } from '@material-ui/icons';
+
+interface IContextFormChipProps {
+    label: string;
+    description?: string;
+    onRemove: () => void;
+}
+
+export const ContextFormChip = ({
+    label,
+    description,
+    onRemove,
+}: IContextFormChipProps) => {
+    const styles = useStyles();
+
+    return (
+        <li className={styles.container}>
+            <div>
+                <div className={styles.label}>{label}</div>
+                {description && (
```

Comment:
I would prefer `ConditionallyRender` here, even though this is a small check. The `&&` pattern can be dangerous (not with this input, but in other contexts - https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) 

If we don't use it in the codebase it will discourage new developers from using this pattern.